smac/smac_cli.py

```diff
@@ -58,13 +59,26 @@ def main_cli(self):
                         in_runhistory_fn_list=args_.warmstart_runhistory,
                         cs=scen.cs,
                         aggregate_func=aggregate_func)
-
+            
+        initial_configs = None
+        if args_.warmstart_trajectory:
+            initial_configs = [scen.cs.get_default_configuration()]
+            for traj_fn in args_.warmstart_trajectory:
+                trajectory = TrajLogger.read_traj_aclib_format(fn=traj_fn, cs=scen.cs)
+                initial_configs.append(trajectory[-1]["incumbent"])
```

Comment:
If this option only reads in the final incumbent of a trajectory is "warmstart_trajectory" the right name for it? Shouldn't it rather be "warmstart_finalIncumbent", "warmstart_Incumbents" or read in the complete trajectory?